job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1738,6 +1738,23 @@ class PublisherContext extends AbstractExtensibleContext {
         }
     }
 
+    /*
```

Comment:
I've already fixed it in the below PR: 
- https://github.com/jenkinsci/job-dsl-plugin/pull/694